test/test.rb

```diff
@@ -34,7 +34,8 @@ class << self
     #Run scan on app at the given path
     def run_scan path, name = nil, opts = {}
       opts.merge! :app_path => "#{TEST_PATH}/apps/#{path}",
-        :url_safe_methods => [:ensure_valid_proto!]
+        :url_safe_methods => [:ensure_valid_proto!],
+        :sql_safe_methods => [:ensure_valid_proto!]
```

Comment:
This shouldn't be tested here. It should be in a specific app's test.

(`url_safe_methods` shouldn't be here either, really.)